xml/deploy_cephadm.xml

```diff
@@ -1902,6 +1902,49 @@ encryption: True
      <xref linkend="deploy-cephadm-day2-orch-status"/>.
     </para>
    </sect3>
+   <sect3 xml:id="cephadm-apply-cluster-specs-">
+    <title>Exporting the Specification of a Running Cluster</title>
+    <para>
+     Although you deployed services to the &ceph; cluster by using the
+     specification files as described in
+     <xref
+      linkend="cephadm-service-and-placement-specs" />, the
+     configuration of the cluster may diverge from the original specification
+     during its operation. Also, you may have removed the specification files
+     accidentally.
+    </para>
+    <para>
+     To retrieve a complete specification of a running cluster, run:
+    </para>
+<screen>
+&prompt.cephuser;ceph orch ls --export
+placement:
+  hosts:
+  - hostname: ses-min1
+    name: ''
+    network: ''
+service_id: my_cephfs
+service_name: mds.my_cephfs
+service_type: mds
+---
+placement:
+  count: 2
+service_name: mgr
+service_type: mgr
+---
+[...]
+</screen>
+    <tip>
+     <para>
+      You can append the <option>--format</option> option to change the default
+      <literal>yaml</literal> output format. You can choose from
+      <literal>json</literal>, <literal>json-pretty</literal>,
+      <literal>xml</literal>, <literal>xml-pretty</literal>, or
+      <literal>plain</literal>. For example:
+     </para>
+<screen>ceph orch ls --export --format xml</screen>
```

Comment:
1. for --export, you have the choice between json-pretty, json and yaml.
2. --format xml is in not supported